src/shared/models/origin.ts

```diff
@@ -0,0 +1,60 @@
+import * as Rt from 'runtypes';
+
+export type LocalOrigin = {
+  type: 'local';
+  filePath: string;
```

Comment:
You will need `fileOffset` here to determine the error location in the file within monokle/validation.